Documentation/learning/api_guarantees.md

```diff
@@ -59,8 +63,14 @@ etcd does not ensure linearizability for watch operations. Users are expected to
 
 etcd ensures linearizability for all other operations by default. Linearizability comes with a cost, however, because linearized requests must go through the Raft consensus process. To obtain lower latencies and higher throughput for read requests, clients can configure a request’s consistency mode to `serializable`, which may access stale data with respect to quorum, but removes the performance penalty of linearized accesses' reliance on live consensus.
 
+### Granting, attaching and revoking leases
+
+etcd provides [the lease mechanism][lease]. The primary usage of lease is implementing distributed coordination mechanisms like distributed locks. The lease mechanism itself is simple: a lease can be created with grant API, attached to a key with put API, revoked with revoke API, and expired by passing of time. However, users need to be aware about [the important properties of the APIs and usage][why] for implementing correct distributed coordination mechanisms.
```

Comment:
"The primary usage of lease" -> "The primary use case of a lease"
"a lease can be created with grant API" -> "a lease can be created with the grant API"
"attached to a key with put API" -> "attached to a key with the put API"
"revoked with revoke API" -> "revoked with the revoke API"
"and expired by passing of time." -> "and will be expired by the wall clock time to live (TTL)."